sopel_modules/rat_board.py

```diff
@@ -1575,7 +1575,7 @@ def ratmama_parse(bot, trigger):
         if result.created:
             # Add IRC formatting to fields, then substitute them into to output to the channel
             # (But only if this is a new case, because we aren't using it otherwise)
-            systemSearch = sysapi_query(fields["system"], "search")
+            validatedSystem = validate_system(fields["system"])
```

Comment:
naming convention, also alarming similarity may lead to avoidable errors..

Reply:
funny enough... I have already tried doing `validatedSystem = validatedsystem(...)` by mistake

😆